tests/dispatch/tests.py

```diff
@@ -96,6 +96,17 @@ def test_garbage_collected(self):
         self.assertEqual(result, [])
         self.assertTestIsClean(a_signal)
 
+    def test_send_robust_logger_exception(self):
```

Comment:
I would move it to the `test_send_robust_fail()`.

Reply:
merged into `test_send_robust_fail()`